storagenode/storagenodedb/bandwidthdb.go

```diff
@@ -34,14 +36,25 @@ func (db *DB) Bandwidth() bandwidth.DB { return db.info.Bandwidth() }
 // Bandwidth returns table for storing bandwidth usage.
 func (db *InfoDB) Bandwidth() bandwidth.DB { return &db.bandwidthdb }
 
+// Run starts the background process for rollups of bandwidth usage
+func (db *bandwidthdb) Run(ctx context.Context) (err error) {
+	defer mon.Task()(&ctx)(&err)
+	return db.loop.Run(ctx, db.Bandwidth().Rollup)
+}
+
+// Close stops the background process for rollups of bandwidth usage
+func (db *bandwidthdb) Close() (err error) {
+	db.loop.Close()
+	return nil
```

Comment:
this is ignoring the error from Close(). Could you do `return db.loop.Close()`

Reply:
Unfortunately, cycle.Close() doesn't return anything

Reply:
ahh was looking at the wrong `Close()` definition